internal/log/package.go

```diff
@@ -0,0 +1,2 @@
+// Logging utility functions
+package log
```

Comment:
Is the nesting necessary?  Couldn't we have `internal/olog` be the package, instead of `internal/log/olog`?

Reply:
Sure thing.